packages/workato-adapter/src/filters/cross_service/jira/project_issuetypes.ts

```diff
@@ -80,6 +110,32 @@ const splitProjectAndIssueType = (
   }
 }
 
+const mergeProjectAndIssueType: ProjectIssuetypeFunc = (value, argName, firstKey, secondKey) => {
```

Comment:
By named params I meant to something like - 

```suggestion
const mergeProjectAndIssueType = ({
  value,
  argName,
  firstKey,
  secondKey,
}: {
  value: JiraExportedBlock,
  argName: PROJECT_ISSUETYPE | SAMPLE_PROJECT_ISSUETYPE,
  ....
}) => {
```

When there's more than 3 inputs it makes code more readable (and helps when there are many params from the same type, but not relevant here)

Reply:
also a naming nit: maybe something like `originalFieldName` instead of `argName` ?

Reply:
I understand, but after I see I use the same args in splitProjectAndIssueType I thought it would be better just make func type, dont you think?